### PR TITLE
Add README.rst and remove description from __openerp__.py

### PR DESCRIPTION
**Pull request by Doscaal**

### CODE REVIEW

**Comment by florian-dacosta** [comment]
remove the dependance from readme file, else :+1:

File: stock_location_ownership/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Stock Location Ownership
+========================
+
+Adds an ownership on the stock locations
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (pedrobaeza) [inline stock_location_ownership/README.rst]: Remove this useless section

**Comment by Doscaal** [comment]
I remove this section from my previous commit.

**Comment by max3903** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**